modules/015-admission-policy-engine/crds/doc-ru-operation-policy.yaml

```
@@ -0,0 +1,86 @@
+spec:
+  versions:
+    - name: v1alpha1
+      schema:
+        openAPIV3Schema:
+          description: |
+            Описывает операционную политику для кластера.
+
+            Каждый ресурс `OperationPolicy` описывает правила для объектов в кластере.
+          properties:
+            spec:
+              properties:
+                policies:
+                  properties:
+                    allowedRepos:
+                      description: |
+                        Список префиксов, которые может иметь образ контейнера.
+                    requiredResources:
+                      description: |
+                        Набор ресурсов, который требуют контейнеры.
+                      properties:
+                        limits:
+                          description: "Список ограничений, которые должны соблюдаться (процессор, память или оба)."
+                        requests:
+                          description: "Список запросов, которые должны быть выполнены (вычислительная мощность, память или и то, и другое)."
+                    disallowedImageTags:
+                      description: "Требует, чтобы образы контейнера имели тег, отличный от указанных в списке."
+                    requiredLabels:
+                      description: |
+                        Список лейблов, которые должен указать объект.
+                      properties:
+                        labels:
+                          items:
+                            properties:
+                              key:
+                                description: >-
+                                  Требуемый лейбл.
+                              allowedRegex:
+                                description: >-
+                                  Если указано, регулярное выражение, которому должно соответствовать значение лейбла
+                                  должно соответствовать. Значение должно содержать хотя бы одно совпадение с
+                                  регулярному выражению.
```

```suggestion
                                  Если указано, то содержит регулярное выражение, которому должно соответствовать значение лейбла. Значение должно содержать хотя бы одно совпадение с регулярным выражением.
```

Not sure I got I right